firmware/common/hackrf_core.c

```diff
@@ -373,6 +373,8 @@ bool sample_rate_frac_set(uint32_t rate_num, uint32_t rate_denom)
 		}
 	}
 
+	SGPIO_GPIO_OUTREG |= (1L << 10); // disable cpld processing
```

Comment:
This should call `sgpio_cpld_stream_disable` instead of poking at the register directly.

Reply:
fixed, i prefer not to check if it was disabled before .. i like to only add ifs when they would make a difference somehow.